modules/swagger-codegen-maven-plugin/src/main/java/io/swagger/codegen/plugin/CodeGenMojo.java

```diff
@@ -217,6 +225,14 @@ public void execute() throws MojoExecutionException {
             configurator.setLibrary(library);
         }
 
+        if(isNotEmpty(modelNamePrefix)) {
+            configurator.setModelNamePrefix(modelNamePrefix);
+        }
+
+        if(isNotEmpty(modelNameSuffix)) {
+            configurator.setModelNamePrefix(modelNameSuffix);
```

Comment:
You have a copy+paste error here, this should be `configurator.setModelNameSuffix(...)`.